jenkins/Jenkinsfile

```diff
@@ -14,18 +14,26 @@ pipeline {
     }
     stage('Build and Test') {
       agent { dockerfile { dir "jenkins" } }
+      environment { GITHUB_TOKEN = credentials('rv-jenkins') }
```

Comment:
Do we need this anymore if we're using the `git` Jenkins command to checkout Firefly repo?

Reply:
If the git jenkins command works without it then we can get rid of it